src/serialization.cpp

```diff
@@ -153,6 +153,11 @@ tensorflow::TensorProto& ProtoGetter<tensorflow::serving::PredictResponse*, tens
     return (*protoStorage->mutable_outputs())[name];
 }
 
+template <>
+::inference::ModelInferResponse::InferOutputTensor& ProtoGetter<::inference::ModelInferResponse*, ::inference::ModelInferResponse::InferOutputTensor&>::get(const std::string& name) {
+    return *protoStorage->add_outputs();
```

Comment:
Why don't we use name parameter here? Is it because in KFS the outputs are not in a map but in a vector?

Reply:
Exactly